### PR TITLE
Add ngx-monetization to tools

### PR DESCRIPTION
**Pull request by CDDelta**
This adds ngx-monetization, an Angular library that lets you use web monetization with RXJS, to the tools section.

### CODE REVIEW

File: website/static/img/angular_webmo_logo.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <g stroke-width="0" id="svg_2">
+            <path d="M210.0005,125.71209999999999 L210.0005,164.80610000000001 " style="fill:none;stroke:black;stroke-width:8.17px;" stroke-width="0" id="svg_3"/>
+        </g>
+        <g stroke-width="0" id="svg_4">
+            <path d="M190.45299999999997,145.25910000000005 L229.547,145.25910000000005 " style="fill:none;stroke:black;stroke-width:8.17px;" stroke-width="0" id="svg_5"/>
+        </g>
+    </g><g id="svg_11" class="">
```
Comment (marcoscaceres) [inline website/static/img/angular_webmo_logo.svg:12]:
```suggestion
    <g id="svg_11" class="">
```

File: website/static/img/angular_webmo_logo.svg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <g stroke-width="0" id="svg_28">
+            <rect x="70" y="55" width="5" height="15" style="fill:rgb(106,218,171);" transform="matrix(1,0,0,1,265.04449999999997,82.75909999999999) " stroke-width="0" id="svg_17"/>
+        </g>
+        <g stroke-width="0" id="svg_30">
+            <rect x="80" y="50" width="5" height="25" style="fill:rgb(106,218,171);" transform="matrix(1,0,0,1,265.04449999999997,82.75909999999999) " stroke-width="0" id="svg_19"/>
+        </g>
+        <g stroke-width="0" id="svg_32">
+            <rect x="90" y="45" width="5" height="35" style="fill:rgb(106,218,171);" transform="matrix(1,0,0,1,265.04449999999997,82.75909999999999) " stroke-width="0" id="svg_21"/>
+        </g>
+    </g></g></svg>
```
Comment (marcoscaceres) [inline website/static/img/angular_webmo_logo.svg:44]:
```suggestion
    </g></svg>
```

**Comment by marcoscaceres** [comment]
The SVG file doesn't seem to be valid XML.

**Comment by CDDelta** [comment]
The file itself seems fine, seems like GitHub's syntax checking doesn't like the same XML tags on the same line. I have formatted the file to have more consistent line breaks. :)

**Pull request merged**